notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -220,6 +220,56 @@ father and son [Sherman Klump had a heart to heart talk with his father Cletus a
 giant hamster []
```

Comment:
That poor empty comment box...

Reply:
We'll get to the bottom of what was going on with that giant hamster one of these days.